android/src/main/java/com/amplitude/android/Configuration.kt

```diff
@@ -1,32 +1,32 @@
 package com.amplitude.android
 
 import android.content.Context
-import com.amplitude.android.events.Plan
 import com.amplitude.android.utilities.AndroidLoggerProvider
 import com.amplitude.android.utilities.AndroidStorageProvider
 import com.amplitude.core.Configuration
 import com.amplitude.core.EventCallBack
 import com.amplitude.core.LoggerProvider
 import com.amplitude.core.ServerZone
 import com.amplitude.core.StorageProvider
+import com.amplitude.core.events.Plan
 
 class Configuration @JvmOverloads constructor(
-    apiKey: String,
+    override val apiKey: String,
```

Comment:
why we use override here for api key?

Reply:
Good catch, that was accidentally added..